### PR TITLE
Fix doc comment about onPressed for IconButton

### PR DESCRIPTION
**Pull request by goderbauer**

### CODE REVIEW

File: packages/flutter/lib/src/material/icon_button.dart [hunk 1]
```diff
@@ -25,8 +25,8 @@ import 'tooltip.dart';
 /// Icon buttons are commonly used in the [AppBar.actions] field, but they can
 /// be used in many other places as well.
 ///
-/// If the [onPressed] callback is not specified or null, then the button will
-/// be disabled, will not react to touch.
+/// If the [onPressed] callback is null, then the button will be disabled, will
```
Comment (abarth) [inline packages/flutter/lib/src/material/icon_button.dart]: "will be disabled and will not react to touch"

**Comment by abarth** [comment]
Do any of the other buttons have this bogus text?

**Comment by goderbauer** [comment]
Yeah, looks like there are more buttons, who have a similar doc comment. Working on including those in this PR.

**Comment by goderbauer** [comment]
Updated all the other buttons that had the bogus text.

**Comment by Hixie** [comment]
LGTM

**Pull request merged**